gobblin-core/src/main/java/gobblin/qualitychecker/row/RowLevelPolicyChecker.java

```diff
@@ -61,6 +68,15 @@ public boolean executePolicies(Object record, RowLevelPolicyCheckResults results
     return true;
   }
 
+  private Path getErrFilePath(RowLevelPolicy policy) {
+    String errFileName = policy.toString().replaceAll("\\.", "-");
```

Comment:
I am not sure why "." is escaped. Regardless, we should also escape ":" and "/" since those are not allowed in HDFS file names. We should probably have a utility function to do that.